sci-libs/clblast/files/patches/level3_xtrsv

```diff
@@ -0,0 +1,11 @@
+--- CLBlast/src/routines/level3/xtrsm.cpp	2022-05-21 08:44:13.815816361 -0600
++++ CLBlast_fixed/src/routines/level3/xtrsm.cpp	2022-05-21 08:28:06.756355738 -0600
```

Comment:
Could you put a comment at the top of each of the patches explaining what it's for (and ideally why it works/how)? Also include any links to upstream submission/bugs?

Reply:
Done.  Pushed new commit.